BTCPayServer/Controllers/UIBoltcardController.cs

```diff
@@ -0,0 +1,70 @@
+#nullable enable
+using Dapper;
+using System.Linq;
+using System.Security;
+using System.Threading.Tasks;
+using BTCPayServer.Data;
+using BTCPayServer.NTag424;
+using BTCPayServer.Services;
+using LNURL;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using System.Threading;
+using System;
+using NBitcoin.DataEncoders;
+using System.Text.Json.Serialization;
+
+namespace BTCPayServer.Controllers;
+
+public class UIBoltcardController : Controller
+{
+    public class BoltcardSettings
+    {
+        [JsonConverter(typeof(NBitcoin.JsonConverters.HexJsonConverter))]
+        public byte[]? IssuerKey { get; set; }
+    }
+    public UIBoltcardController(
+        UILNURLController lnUrlController,
+        SettingsRepository settingsRepository,
+        ApplicationDbContextFactory contextFactory,
+        BTCPayServerEnvironment env)
+    {
+        LNURLController = lnUrlController;
+        SettingsRepository = settingsRepository;
+        ContextFactory = contextFactory;
+        Env = env;
+    }
+
+    public UILNURLController LNURLController { get; }
+    public SettingsRepository SettingsRepository { get; }
+    public ApplicationDbContextFactory ContextFactory { get; }
+    public BTCPayServerEnvironment Env { get; }
+
+    [AllowAnonymous]
+    [HttpGet("~/bc")]
```

Comment:
Imho this is too concise :)

```suggestion
    [HttpGet("~/boltcard")]
```